autodoc.pl

```diff
@@ -534,7 +533,19 @@ ($$$$)
                         # apidoc_item and first char of arg is '|'
                     && (   length $9 != 0
                         || (   $type == APIDOC_ITEM
-                            && substr($10, 0, 1) eq '|'));
+                            && substr($10, 0, 1) eq '|')));
+        if ($mostly_proper_form) {
+            return ($type, $arg) if length $1 == 0; # Is fully correct if left
+                                                    # justified
+
+            # Ordinarily not being left justified is an error, but special
+            # case perlguts which has examples of how to use apidoc in
+            # verbatim blocks, which we don't want to confuse with a real
+            # instance and reject because it isn't left justified.  Use a
+            # precise indent to get the precise lines that have this.
+            return (NOT_APIDOC, $input) if length $1 == 1
+                                        && $file eq 'pod/perlguts.pod';
```

Comment:
Shouldn't this just exclude any `.pod`?

Reply:
No; that would miss some typos.  We should only document how to use this in one place; that place is in perlguts now.  This is an imperfect heuristic, with tension between not wanting to make it so that an insignificant change to the line in perlguts would cause it to fail, and not wanting to make it so loose that it misses introduced typos in that or any pod